components/_previews/_buttons-in-alerts/buttons-in-alerts.hbs

```diff
@@ -0,0 +1,63 @@
+<div class="dso-example-wrapper">
+  <div class="dso-example-text">
+    <h2>Buttons in sectieberichten</h2>
+  </div>
+  <div class="dso-example">
+    <table class="table">
+      <caption>Verschijningsvormen van buttons in sectieberichten (<code>.alert</code>).</caption>
+      <thead>
+        <tr>
+          <th scope="col">Beschrijving</th>
+          <th scope="col">Weergave</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Waarschuwing</td>
+          <td>
+            <div class="alert alert-warning" role="alert">
+              <div class="dso-rich-content">
+                <p>Dit is een waarschuwingsmelding. Deze wordt gebruikt voor waarschuwingen.</p>
+                {{> '@button' type='button' label='Zoekopdracht afbreken' }}
```

Comment:
De `button` kan niet direct in een `dso-rich-content`, maar moet in een `p` staan

Reply:
done